src/views/dashboard.ejs

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+
+    <!-- Bootstrap CSS -->
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC"
+      crossorigin="anonymous"
+    />
+
+    <style>
+      html {
+        min-height: 100vh;
+      }
+      .container {
+        height: 10%;
+      }
+      body {
+        height: 100vh;
+      }
+      .card-container {
+        display: flex;
+        justify-content: space-around;
+        align-items: center;
+        flex-direction: column;
+        width: 100%;
+        height: 80%;
+      }
+    </style>
+
+    <title>Dashboard</title>
+  </head>
+  <body>
+    <div class="container mt-4" style="text-align: center">
+      <h1>Registration Dashboard</h1>
+    </div>
+    <div class="card-container">
+      <div class="card">
+        <!-- <img src="..." class="card-img-top" alt="..."> -->
+        <div class="card-body">
+          <h5 class="card-title">Stand Registration</h5>
+          <p class="card-text">Click here for stand registration</p>
+          <a href="/register/stand" class="btn btn-primary">register stand</a>
```

Comment:
I didn't test this code, but are all three going to be displayed? If so, it seems redundant, just the link with the relevant description can be there. Similarly for the other cards.